pkgs/development/libraries/xtensor/default.nix

```diff
@@ -2,36 +2,38 @@
 , stdenv
 , fetchFromGitHub
 , cmake
-, gtest
+, doctest
 , xsimd
 , xtl
 }:
 stdenv.mkDerivation rec {
   pname = "xtensor";
-  version = "0.23.10";
+  version = "0.24.6";
 
   src = fetchFromGitHub {
     owner = "xtensor-stack";
     repo = "xtensor";
     rev = version;
-    sha256 = "1ayrhyh9x33b87ic01b4jzxc8x27yxpxzya5x54ikazvz8p71n14";
+    sha256 = "sha256-cTzQSXn9REp/sPCMmBFJkTW0GS2CFOjPvgGlYG6pxT0=";
   };
 
+  # https://github.com/xtensor-stack/xtensor/issues/2542
+  postPatch = ''
+    substituteInPlace xtensor.pc.in \
+      --replace '$'{prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@
+  '';
+
   nativeBuildInputs = [ cmake ];
   propagatedBuildInputs = [ xtl xsimd ];
 
   cmakeFlags = [ "-DBUILD_TESTS=ON" ];
 
   doCheck = true;
-  nativeCheckInputs = [ gtest ];
+  checkInputs = [
```

Comment:
I think `checkInputs` is discouraged mostly, ever since https://github.com/NixOS/nixpkgs/pull/206742/commits/f39abbc3500451fc00c80fed71b0f6f7bba8b6a4 . Use `nativeCheckInputs`, when cross compiling, testing doesn't make sense usually.